### ISSUE TITLE
Community chat

### ISSUE THREAD

**Issue by eshrh** [jrnl-org/jrnl#1165]
## Feature Request
I was just thinking that it might be nice to have some real-time chat centered around jrnl. This might be through irc, matrix, slack or whatever else the maintainers like. I'm fine with this being only for development discussion as well (to avoid it becoming a pure support forum). 

### Use Case/Motivation
I think this could be useful when talking about future ideas or complex development plans. Of course, I imagine it would also be pretty entertaining. It would be a nice way to increase community involvement, and have a conversation instead of a back-and-forth of long posts in a github issue.

The downside is that although these things are nice, they might need moderation which the maintainers aren't willing or able to do. Perhaps other community members might step in for this...

**Comment by KarimPwnz** [comment]
I would love this! Nevertheless, we should ensure a GitHub log of decisions/discussions made elsewhere. I suggest we include summaries on GitHub.

**Comment by wren** [comment]
Thanks for the suggestion! This has come up before (don't remember which issue it was in), and I'm not against it.

I'm actually generally for it, but acknowledge that I don't have the time to properly maintain something like this. But if we can get a volunteer or three to moderate (and commit to upholding the code of conduct), I'd be happy to get something set up and linked in the readme.

To be clear, though, any forum like this would be only for conversation. If and when a conversation leads to something actionable, then I would still want an issue to be filed here with enough details required to complete the task (@micahellison and I have regularly scheduled sessions where we go over issues, and backlog grooming, and all that other fun stuff, and that process would break down if we had to review chatlogs to organize issues).

I'll leave this open for a while in case someone wants to step up to moderate

**Comment by KarimPwnz** [comment]
> I'm actually generally for it, but acknowledge that I don't have the time to properly maintain something like this. But if we can get a volunteer or three to moderate (and commit to upholding the code of conduct), I'd be happy to get something set up and linked in the readme.

I'd be down to moderate!

> To be clear, though, any forum like this would be only for conversation. If and when a conversation leads to something actionable, then I would still want an issue to be filed here with enough details required to complete the task (@micahellison and I have regularly scheduled sessions where we go over issues, and backlog grooming, and all that other fun stuff, and that process would break down if we had to review chatlogs to organize issues).

100%

**Comment by micahellison** [comment]
Alright! We set up a Gitter chat here: [![Gitter](https://badges.gitter.im/jrnl-org/jrnl.svg)](https://gitter.im/jrnl-org/jrnl?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)

Gitter looks ideal since it has GitHub integration and is completely open and searchable. I think it will also be nice to be able to use the same markdown that we use here on GitHub.

I've also added this chat badge to the README as well.

For now, I want us to be open to any kind of discussion related to jrnl, whether it's support or development. But if those two functions get too hard to manage in one room, we can always create a second room.
